user/pages/05.blog/2017-08-15-cpu-monitoring/item.md

```diff
@@ -0,0 +1,54 @@
+---
+title: CPU monitoring
+date: 21:51 2017-08-15
+author: Marcel Fuhrmann
+body_classes: header-lite fullwidth blogstyling
+taxonomy:
+    category: wiki
+    tag: [configuration, thresholding]
+---
+
+Oh really? What's special about it?
+
+Yes... Ehm.. Not much.
+But I'd like to talk about it because I think it could be useful.
+
+In this times you usually have enough CPU performance and often you don't have to care about it.
+And that's good.
+I never had a really critical situation with high CPU usage.
+Maybe I had luck.
+
+Sure, sometimes some processes freak out and you want get informed when CPU is running on limit.
+But that's not all.
+
+Imagine that you have your own software product and you deploy daily the newest version.
+Every day new code runs in your system.
+So your environment is daily an other one then yesterday.
+
+Things can happen...
+
+...also your code can work different and maybe it eating more and more CPU.
+
+But just small steps.
+
+It's not extreme enough to make noise and trigger usual alarms like >90% etc.
+
+
+So what do we need?
+
+Figure out what's your usual CPU usage.
+Often the usage is really straight.
+Let's say 30% constantly while working hour.
+So you need a threshold like above 40% constantly for a long time.
+1 hour for example.
```

Comment:
For numbers from 1-12, you usually write them down in letters instead of numbers. At least, in this case, I would prefer it.